docs/userguide/networking/dockernetworks.md

```diff
@@ -404,16 +404,27 @@ provision the hosts are with Docker Machine.
 
 ![Engine on each host](images/engine_on_net.png)
 
+It is important to make sure that the following ports are open between each 
+of your hosts:
+
+| Protocol | Port | Description           |
+|----------|------|-----------------------|
+| udp      | 4789 | VXLAN                 |
+| tcp/udp  | 7946 | Overlay control plane |
+
+You should also ensure that any ports required by your key-value store service
+are open.
+
 Once you have several machines provisioned, you can use Docker Swarm to quickly
 form them into a swarm which includes a discovery service as well.
 
 To create an overlay network, you configure options on  the `daemon` on each
 Docker Engine for use with `overlay` network. There are two options to set:
 
-| Option                           | Description                                               |
-|----------------------------------|-----------------------------------------------------------|
-| `--cluster-store=PROVIDER://URL` | Describes the location of the KV service.               |
-| `--cluster-advertise=HOST_IP`    | Advertises containers created by the HOST on the network. |
+| Option                                        | Description                                               |
+|-----------------------------------------------|-----------------------------------------------------------|
+| `--cluster-store=PROVIDER://URL`              | Describes the location of the KV service.                 |
+| `--cluster-advertise=HOST_IP|HOST_IFACE:PORT` | Advertises containers created by the HOST on the network. |
```

Comment:
the description is incorrect : `Advertises containers created by the HOST on the network`
It must be `Advertises the host reachability in the cluster`

I understand this comment doesnt belong to this PR.
But I think it is better to fix it while we are at it.